docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -692,6 +692,72 @@
       process by calling <emphasis>DFUCreateFile</emphasis> and publishing the
       file once it has been written by calling
       <emphasis>DFUPublishFile</emphasis>.</para>
+
+      <sect2 role="brk">
+        <title>Examples using Primary Classes</title>
```

Comment:
@JamesDeFabia Could we change the title here to something along the lines of: "Using Spark Datasource API to Read and Write". Otherwise looks good to me

Reply:
@jpmcmu Made the change, please review again